interface/main/tabs/templates/menu_template.php

```diff
@@ -16,12 +16,12 @@
 
 <script type="text/html" id="menu-action">
     <i data-bind="css: icon,text:helperText" class="fa closeButton"></i>
-    <div class='menuLabel px-1' data-bind="text:label,click: menuActionClick,css: {menuDisabled: ! enabled()}"></div>
+    <div class='menuLabel pr-md-3 pr-sm-1 pr-xl-4' data-bind="text:label,click: menuActionClick,css: {menuDisabled: ! enabled()}"></div>
```

Comment:
How do the changes in this script look without the icons? (just thinking may be good to at least have code in here to make the icons looks good even if we do not include them in the official json menus (note it is very easy to create custom menus and assign them to users)

Reply:
We discussed on call that we have global to turn on icons only or default labels as currently styled.

Reply:
@bradymiller I will see what the menu looks like without the icons added. I will post the screenshot here.